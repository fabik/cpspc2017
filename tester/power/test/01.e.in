99
815 111
6 579
250 933
336 972
555 3
12 389
990 405
55 270
790 907
6 420
106 190
460 998
492 0
71 756
617 985
995 563
586 992
197 102
397 989
839 867
678 32
736 59
920 229
30 328
964 685
149 143
975 343
648 22
88 783
902 203
127 833
964 314
41 299
249 66
30 671
428 5
0 515
586 7
366 981
995 436
492 999
149 856
88 216
428 994
648 977
983 374
990 594
223 83
936 743
172 122
2 452
936 256
861 845
12 610
197 897
41 700
678 967
523 999
883 178
707 45
763 75
998 468
523 0
861 154
223 916
951 714
0 484
763 924
460 1
277 52
951 285
839 132
336 27
71 243
790 92
983 625
902 796
55 729
975 656
617 14
306 38
397 10
277 947
707 954
998 531
920 770
1000 500
736 940
2 547
172 877
106 809
20 359
127 166
883 821
555 996
20 640
306 961
366 18
815 888
