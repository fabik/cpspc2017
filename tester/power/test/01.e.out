1066
50
1 2 3 6 7 8 9 14 15 20 24 27 28 30 37 38 39 41 42 43 44 47 48 50 51 52 55 56 57 60 61 62 63 64 66 69 70 71 73 79 82 85 86 88 91 93 94 95 96 97
49
4 5 10 11 12 13 16 17 18 19 21 22 23 25 26 29 31 32 33 34 35 36 40 45 46 49 53 54 58 59 65 67 68 72 74 75 76 77 78 80 81 83 84 87 89 90 92 98 99
